docker/ubuntu-latest-amd64-openssl-3.0.x/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:latest
+
+ARG NODE_VERSION="18.12.1"
```

Comment:
We discussed that this could use the most recent LTS version and that we could have a test for the latest version too